ember_debug/models/profile-manager.js

```diff
@@ -80,6 +156,59 @@ export default class ProfileManager {
     }
   }
 
+  teardown() {
+    this.stylesheet.remove();
+    // remove all the active highlighted components
+    this.removeAllHighlights();
+  }
+
+  removeAllHighlights() {
+    const els = this.highlights.slice(0);
+    els.forEach((el) => {
+      this.removeHighlight(el);
+    });
+  }
+
+  removeHighlight(el) {
+    this.highlights = this.highlights.filter((item) => item !== el);
+    clearTimeout(el.timeout);
+    el.node.remove();
+  }
+
+  addHighlight(highlight) {
+    this.highlights.push(highlight);
+
```

Comment:
and likely here, we should call `_insertHighlight` method, to inject item into DOM

Reply:
updated.